app/src/main/res/layout/activity_create_config.xml

```diff
@@ -47,7 +47,7 @@
             <LinearLayout
                 android:layout_width="match_parent"
                 android:layout_height="@dimen/height_zero"
-                android:layout_weight="@dimen/weight_one"
+                android:layout_weight="@dimen/weight_two"
```

Comment:
Is it necessary to change weight ??? and add another linear layout to embed the recycler view and all the stuff.